localization/english/event_localization/wc_horde_invasion_events_l_english.yml

```diff
@@ -0,0 +1,23 @@
+ wc_horde_invasion.0001.desc:0 "As the defenses of $c_northshire$ and its surrounding lands fall, our warriors descend into a frenzy of destruction. The breakthrough has unleashed an orgy of violence; every building is pillaged, every human cut down in their tracks.\n\nThe final stand of the humans is at the abbey. As our warriors surround it, we uproot an ancient tree from the gardens to use as a battering ram. Each thunderous crash against the abbey's doors mingles the roars of our war cries with the terrified screams of those inside. The timber groans, splinters flying, as the last barrier to our wrath buckles and bends.\n\nJust as the door begins to give way, poised to unleash our horde into their last haven, we prepare to extinguish the final flicker of resistance in $c_northshire$."
+ wc_horde_invasion.0001.a:0 "Lok'tar Ogar, for the Horde!"
+
+ wc_horde_invasion.0002.title:0 "The Last Stand of Northshire"
+ wc_horde_invasion.0002.desc:0 "The fall of $c_northshire$ is a nightmare come to life. The lands burn around us, and our people are mercilessly slaughtered. We last few defenders have retreated to the abbey, our final sanctuary.\n\nThrough the barricaded doors, we hear the orcs' fury grow louder. They have uprooted one of the sacred Holyoak trees from our gardens, transforming it into a tool of destruction. Each impact against the doors sends a wave of dread through our hearts. Our prayers for salvation are drowned out by the cacophony of battle - the orcs' battle cries, the splintering wood, the screams of the doomed.\n\nAs the doors begin to splinter and crack, the end nears. We brace ourselves, knowing that the flood of rage and steel is about to break through, and with it, the last light of $c_northshire$ will be extinguished."
+ wc_horde_invasion.0002.a:0 "For Northshire! For $k_stormwind$!"
+ wc_horde_invasion.0002.tooltip.a:0 "Faced with the invader, we resist! Let us fight with honor for our land, for Northshire! For $k_stormwind$!"
+
+ wc_horde_invasion.0003.title:0 "The Sack of Northshire"
+ wc_horde_invasion.0003.desc:0 "Now that Northshire is under our control, bands of orcs spread through the surrounding lands, leaving behind smoking ruins. Homes are looted and burned, fields destroyed, and no inhabitant is spared from their bloodthirsty rage. Faced with this, I am confronted with a choice:\n\n On the one hand, I can let my warriors' thirst for violence run free. Northshire would be devastated, and its riches would fuel the machine of our conquests, a brutal testament to the futility of opposing the Horde.\nAlternatively, I could order my warriors to cease their destruction. Executing those who defy this order would be necessary to maintain discipline. This would allow for the preservation and repurposing of key structures for the use of the Horde, transforming the remains of $c_northshire$ into a strategic foothold for our future campaigns. The decision rests with me..."
```

Comment:
```suggestion
 wc_horde_invasion.0003.title:0 "The Sack of $c_northshire$"
 wc_horde_invasion.0003.desc:0 "Now that $c_northshire$ is under our control, bands of orcs spread through the surrounding lands, leaving behind smoking ruins. Homes are looted and burned, fields destroyed, and no inhabitant is spared from their bloodthirsty rage. Faced with this, I am confronted with a choice:\n\n On the one hand, I can let my warriors' thirst for violence run free. Northshire would be devastated, and its riches would fuel the machine of our conquests, a brutal testament to the futility of opposing the Horde.\nAlternatively, I could order my warriors to cease their destruction. Executing those who defy this order would be necessary to maintain discipline. This would allow for the preservation and repurposing of key structures for the use of the Horde, transforming the remains of $c_northshire$ into a strategic foothold for our future campaigns. The decision rests with me..."
```